frontend/components/MapView.jsx

```diff
@@ -48,6 +71,8 @@ export default function ProjectMap({ projectIds = [], setProjectCount }) {
   const [showCapsDropdown, setShowCapsDropdown] = useState(false);
   const [selectedCapsRanges, setSelectedCapsRanges] = useState([]);
 
+  const pageSize = 10;
```

Comment:
The problem here is that not all projects appear on the map as a result of this. If this is a bit complex to solve right now (which I suspect it is?) can we just put this as a very high number for now (250?) and deal with it when they go over or performance drops too far? We could paginate on the frontend only as I would imagine the api call itself is not the issue - it's react performance for 250 projects being rendered at once. Even that frontend only pagination is not necessary for now though unless it's easy.

Reply:
For 250 projects, I don't think it will impact performance. I believe we should avoid adding pagination for now, since we need to display all projects on the map when it first loads. However, the list view should still have pagination. I've set the page size to 250 for now.

Reply:
Okay, that looks good. Let me know when the list view pagination is done or if you plan to leave this for now.